src/compilers/dbg.cr

```diff
@@ -8,20 +8,34 @@ module Mint
         var =
           [Variable.new] of Item
 
+        arg =
+          if node.bang?
+            var
+          else
+            js.call(Builtin::Inspect, [var])
+          end
+
+        location =
+          if config.generate_source_maps
+            [] of Item
+          else
+            js.call(["console.log"] of Item, [location])
+          end
+
         if expression = node.expression
           js.iif do
             js.statements([
               js.const(var, compile(expression)),
-              js.call(["console.log"] of Item, [location]),
-              js.call(["console.log"] of Item, [var]),
+              location,
+              js.call(["console.log"] of Item, [arg]),
```

Comment:
Why not log `location` in the same `console.log` call as the inspected expression?

```suggestion
              js.call(["console.log"] of Item, [location, arg]),
```

Reply:
They will be printed on the same line:

![image](https://github.com/user-attachments/assets/82d0216d-9fea-407d-9cbf-530aa36f6cd5)